<launch>

  <!-- realsense -->
  <include file="$(find realsense2_camera)/launch/rs_aligned_depth.launch"></include>

  <!-- state machine node-->
  <node pkg="perception_refactor"  name="state_machine_node" type="state_machine_node.py"></node>

  <!-- planner node-->
  <node pkg="perception_refactor"  name="planner_node" type="planner_node.py"></node>

  <!-- visual servo node-->
  <node pkg="perception_refactor"  name="visual_servo_node" type="visual_servo_node.py"></node>

  <!-- ee node-->
  <node pkg="perception_refactor"  name="ee_node" type="ee_node.py"></node>

  <!-- perception node-->
  <node pkg="perception_refactor"  name="perception_node" type="perception_node.py"></node>

</launch>